string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,8 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  if (word === word[0].toUpperCase() + word.substring(1)) {
+    return false;
+  } else if (word.toLowerCase()) {
+    return true;
+  }
+}
```

Comment:
We can simplify this! You don't need all the extra `else..if` statements. A conditional expression will always evaluate to either `true` or `false`, so you don't need to explicitly check for the other cases. We can just do something like this

```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```